Fix host networking in several charts

allen-4 opened the pull request:
**Description**
<!--
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
-->
There is a typo in the variable for hostNetwork on both of these charts. Removed "ing" from questions.yaml and behavior works as expected. Not sure if I should have or not but I bumped the version to 1.4.1 as well.


**Type of change**

- [ ] Feature/App addition
- [x] Bugfix
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor of current code

**How Has This Been Tested?**
<!--
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
-->
Tested on the current 21.02 alpha

**Notes:**
<!-- Please enter any other relevant information here -->

**Checklist:**

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests to this description that prove my fix is effective or that my feature works

allen-4 committed:
Fix host networking in radarr

allen-4 committed:
Fix host networking in radarr

allen-4 committed:
fix nzbhydra networking

allen-4 committed:
fix nzbhydra networking

allen-4 committed:
fix update

Ornias1993 commented:
@allen-4 Thanks for the quick fix, the current version is 1.5.0...
you might want to rebase.

Also, I see there are another 10 with the same issue, would you mind a quick "search and replace" on all of those to fix this everywhere? Because it is indeed wrong.

Note: We do not provide active support for this specific feature on most charts, though it should at least do what it says on the tin.

allen-4 committed:
Fix hostNetwork across 23 charts

allen-4 committed:
Merge branch 'master' of https://github.com/allen-4/charts

allen-4 committed:
Remove old files

allen-4 committed:
Fix merge conflict

allen-4 commented:
I rebased from master and updated the PR. I found 23 total charts with the same problem. I appreciate the disclaimer as I'm new to helm charts. Is there a different way I should be going about enabling external networking on the apps? Documentation is scarce at the moment so I haven't been able to figure out where to look.

Ornias1993 commented:
@allen-4 In general we currently have only one Apps that "needs" this to be turned on and one that might or might-not benefid from heaving it turned on (generally not).

Connecting it to the outside is done using either a nodePort or Reverse Proxy configuration...

Crow-Control reviewed (approved):
Clean fix, without un-intended changes...
Thank you!

Ornias1993 commented:
@all-contributors please add @allen-4 for code

truecharts-admin commented:
This PR is locked to prevent necro-posting on closed PRs. Please create a issue or contact staff on discord if you want to further discuss this
